Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import ArgumentParser
+import TuistSupport
+
+public struct TuistCommand: ParsableCommand {
+    public init() { }
+    
+    public static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "tuist",
+                             abstract: "Generate, build and test your Xcode projects.",
+                             subcommands: [GenerateCommand.self])
+    }
+    
+    public static func main(_ arguments: [String]? = nil) -> Never {
+        let errorHandler = ErrorHandler()
+        let command: ParsableCommand
+        do {
+             command = try parseAsRoot(processArguments(arguments))
+        } catch {
+            logger.error("\(fullMessage(for: error))")
+            _exit(exitCode(for: error).rawValue)
```

Comment:
Where is `_exit` defined?

Reply:
It's defined in `Darwin`- I have tried to copy `parseOrExit(_ arguments: [String]? = nil)` defined in the parser library that we'd use if we did not need custom error handling